Assets/MixedRealityToolkit.SDK/Features/UX/Meshes/Cursors/Cursor_Focus_geo.fbx.meta

```diff
@@ -0,0 +1,108 @@
+fileFormatVersion: 2
+guid: 3f1525ceb67261b429bc8741e7cee099
+ModelImporter:
+  serializedVersion: 23
+  fileIDToRecycleName:
+    100000: //RootNode
+    400000: //RootNode
+    2100000: Light
+    2100002: Shadow
+    2300000: //RootNode
+    3300000: //RootNode
+    4300000: Cursor_Focus
+  externalObjects:
+  - first:
+      type: UnityEngine:Material
+      assembly: UnityEngine.CoreModule
+      name: Light
+    second: {fileID: 2100000, guid: d7c7db2cf580ab94084f0561d16ded07, type: 2}
+  - first:
+      type: UnityEngine:Material
+      assembly: UnityEngine.CoreModule
+      name: Shadow
+    second: {fileID: 2100000, guid: fc1b6cd8da653e3448dcdb722ada2ebc, type: 2}
+  materials:
+    importMaterials: 1
```

Comment:
We should probably turn off `importMaterials` for these models, since this PR is also including its own materials and prefabs.

Reply:
Updated. Thanks!